source/tools.txt

```diff
@@ -1,168 +1,263 @@
-* `Log4Mongo <https://github.com/log4mongo/log4mongo-python>`_ is a flexible
-  Python logging handler that can store logs in MongoDB using normal and capped
-  collections.
-* `MongoLog <http://github.com/puentesarrin/mongodb-log/>`_ is a Python logging
-  handler that stores logs in MongoDB using a capped collection.
-* `rod.recipe.mongodb <http://pypi.python.org/pypi/rod.recipe.mongodb/>`_ is a
-  ZC Buildout recipe for downloading and installing MongoDB.
-* `mongobox <http://github.com/theorm/mongobox>`_ is a tool to run a sandboxed
-  MongoDB instance from within a python app.
-* `Flask-MongoAlchemy <http://github.com/cobrateam/flask-mongoalchemy/>`_ Add
-  Flask support for MongoDB using MongoAlchemy.
-* `Flask-MongoKit <http://github.com/jarus/flask-mongokit/>`_ Flask extension
+- `Django MongoDB Engine <https://django-mongodb-engine.readthedocs.io/en/latest/>`__
+  is a MongoDB database backend for Django that completely integrates with its ORM.
+  For more information, see the `Django MongoDB Engine tutorial
+  <https://django-mongodb-engine.readthedocs.io/en/latest/tutorial.html>`__.
+
+- `Django MongoEngine <https://github.com/MongoEngine/django-mongoengine>`__ is a
+  MongoDB backend for
+  Django. The repository also includes an `example application <https://github.com/MongoEngine/django-mongoengine/tree/master/example/tumblelog>`__.
+  For more information, see the `Django MongoEngine documentation <https://django-mongoengine.readthedocs.io/en/latest/>`__.
+
+- `Djongo <https://www.djongomapper.com/>`__ is a connector for using
+  Django with MongoDB as the database backend. With Djongo, you can use the Django Admin
+  GUI to add and modify documents in MongoDB.
+  The `Djongo source code <https://github.com/doableware/djongo>`__ is hosted on GitHub,
+  and the `Djongo package <https://pypi.python.org/pypi/djongo>`__ is on PyPI.
+
+- `Flask-MongoAlchemy <http://github.com/cobrateam/flask-mongoalchemy/>`__ adds
+  Flask support for MongoDB with MongoAlchemy.
+
+- `Flask-MongoKit <http://github.com/jarus/flask-mongokit/>`__ is a Flask extension
   to better integrate MongoKit into Flask.
-* `Flask-PyMongo <http://github.com/dcrosta/flask-pymongo/>`_ Flask-PyMongo
-  bridges Flask and PyMongo.
+
+- `Flask-PyMongo <http://github.com/dcrosta/flask-pymongo/>`__ bridges Flask and PyMongo.
+
+- `Log4Mongo <https://github.com/log4mongo/log4mongo-python>`__ is a flexible
+  Python logging handler that can store logs in MongoDB by using normal and capped
+  collections.
+
+- `mango <http://github.com/vpulim/mango>`__ provides MongoDB backends for
+  Django sessions and authentication (bypassing ``django.db`` entirely).
+
+- `mongobox <http://github.com/theorm/mongobox>`__ is a tool to run a sandboxed
+  MongoDB instance from within a Python app.
+
+- `mongodb_beaker <http://pypi.python.org/pypi/mongodb_beaker>`__ enables you to use MongoDB
+  as a backend for `beaker's <https://beaker.readthedocs.io/en/latest/>`__
```

Comment:
Minor tweak: I think in this instance you're using the proper name of the tool, so it should be capitalized.
```suggestion
  as a backend for `Beaker's <https://beaker.readthedocs.io/en/latest/>`__
```